### PR TITLE
 Appwrite + Preact blog

### PR DESCRIPTION
**Pull request by Sooraj-s-98**
## What does this PR do?

A simple blog app built with Appwrite and Preact

## Test Plan

https://user-images.githubusercontent.com/52352285/197349002-ea649da2-d1a3-449d-be55-f0d04d65cfb4.mov
 
[Social](https://twitter.com/soorajsdevto)

## Related PRs and Issues

[issue ](https://github.com/appwrite/appwrite/issues/4252)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

yes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +243,9 @@ Share your apps here! Submit a pull request!
 ### Golang
 * **GoRest** A demo app to showcase integrating appwrite apis with golang. Using database endpoints and examples [Source Code](https://github.com/maliaga-pantoja/golang-appwrite-demo)
 
+### PReact
```
Comment (Meldiron) [inline README.md:246]:
```suggestion
### Preact
```

**Review by Meldiron — CHANGES_REQUESTED** [review]
Let's please do this tiny change, other than that, looks good to me.

File: README.md (hunk 1 above)
Comment (Sooraj-s-98) [reply README.md]: sure

**Review by Meldiron — APPROVED** [review]
Looks great 🎉

**Comment by christyjacob4** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swags! As a way of saying thank you, we would also love to invite you to join the Appwrite organization on GitHub. Please share your GitHub username with us on Discord.  

You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Please feel free to look for more PRs you might be interested in helping with on our long list of [Hacktoberfest friendly issues](https://github.com/search?q=org%3Aappwrite+org%3Autopia-php+org%3Aopen-runtimes+label%3Ahacktoberfest+created%3A%3E2021-01-01+state%3Aopen&type=issues&s=updated&o=asc) and help make Appwrite better :)

**Pull request merged**